devops/yarn/checkstyle.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0"?>
```

Comment:
So why we need the checkstyle.xml here, why not use the checkstyle.xml under the root java package?